etc/exceptions-inclusions.csv

```diff
@@ -0,0 +1 @@
+DocId,Namespace,Type,Member,linux,osx,win
```

Comment:
What is this for? Is the idea that we can add exceptions here that the tool can't find?

Reply:
That's the idea, since I was adding exclusion it felt natural to add inclusion too, but so far I don't have any case to include.